landing.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
An item of note: is it possible to place a <script> element in the head of an HTML document? I thought it was best at the very end of the body so that it would only run once the page's other elements had been loaded into the DOM.